docs/quick-start/environment.mdx

```diff
@@ -39,8 +50,12 @@ Visit your app on `http://localhost:3000`. You can interact with your smart cont
 
 ## What's Next:
 
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
-- Edit your deployment scripts in `packages/hardhat/deploy`
+- Edit your smart contract:
+  - Hardhat => `YourContract.sol` in `packages/hardhat/contracts`
+  - Foundry => `YourContract.sol` in `packages/foundry/contracts`
+- Edit your deployment scripts:
+  - Hardhat => `packages/hardhat/deploy`
+  - Foundry => `packages/foundry/script`
 - Edit your frontend in `packages/nextjs/pages`
 - Edit the app config in `packages/nextjs/scaffold.config.ts`
 - Run smart contract test with `yarn hardhat:test`
```

Comment:
Maybe point to `packages/hardhat/test` and use `yarn hardhat:test`  similarly for foundry `packages/foundry/test` => `yarn foundry:test`

Reply:
What looks better for you?

**Option 1.**
- Edit your smart contract test and run it:
  - Hardhat => `packages/hardhat/test` | `yarn hardhat:test`
  - Foundry => `packages/foundry/test` | `yarn foundry:test`

**Option 2.**
- Edit your smart contract test in:
  - Hardhat => `packages/hardhat/test`
  - Foundry => `packages/foundry/test`
- Run smart contract test with:
  - Hardhat => `yarn hardhat:test`
  - Foundry => `yarn foundry:test`

Reply:
For now I've implemented Option 2, but only to not do a push with both of them included 😊 If you like 1 better I'll update it, np!

Reply:
lol confused both look good what if merge them ? : 
```
- Edit your smart contract test in:
  - Hardhat => `packages/hardhat/test` to run test use `yarn hardhat:test`
  - Foundry => `packages/foundry/test` to run test use `yarn foundry:test`
```

But even option 2 looks completely fine !! Please feel free to ignore this 🙌

Reply:
I've just pushed your final version 😊